src/main/java/edu/kis/powp/jobs2d/events/SelectHistoryListOptionListener.java

```diff
@@ -0,0 +1,31 @@
+package edu.kis.powp.jobs2d.events;
+
+import javax.swing.JList;
+import javax.swing.event.ListSelectionEvent;
+import javax.swing.event.ListSelectionListener;
+
+import edu.kis.powp.jobs2d.command.history.HistoryEntry;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+
+public class SelectHistoryListOptionListener implements ListSelectionListener {
+
+    private DriverCommandManager commandManager;
+    private final JList<HistoryEntry> historyCommandList;
```

Comment:
I think that the listener could be independent of JList implementation but rather relying only on ListModel.

Reply:
We have chosen JList because of the ability of selecting elements from list using mouse.

Reply:
Yes, but internally JList relies on ListModel as a data holder.